### ISSUE TITLE
Accessibility: MAS42A: High Contrast: The download arrows are missing for the links in High contrast white theme

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#15]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Navigate to the package download links below the "Get started now " above the footer of the page.
3. Observe the download arrows for the "windows", ".deb", ".rpm", "Mac" download links.

**Actual Result:**
The download arrows are missing for the "windows", ".deb", ".rpm", "Mac" package download links below the "Get started now".

**Expected Result:**
Download arrows  should be visible for the links "windows", ".deb", ".rpm", "Mac"  in high contrast white  theme as visible in normal mode.

**User Impact:**
The persons with colour blindness and low vision will not be able to understand that there are download arrows for the download links and may not use the application effeciently.

**Screenshots:**
[Download arrows not vsible.zip](https://github.com/wiproaccess/vscode-website/files/687091/Download.arrows.not.vsible.zip)

**Comment by waderyan** [comment]
Fixed with 2b4e904688206594e2c4cbcb33974566b74459ff
